opencti-platform/opencti-graphql/tests/03-streams/00-Raw/raw-test.js

```diff
@@ -80,13 +80,13 @@ describe('Raw streams tests', () => {
       expect(updateEventsByTypes['note'].length).toBe(3);
       expect(updateEventsByTypes['opinion'].length).toBe(6);
       expect(updateEventsByTypes['report'].length).toBe(19);
-      expect(updateEventsByTypes['ipv4-addr'].length).toBe(5);
+      expect(updateEventsByTypes['ipv4-addr'].length).toBe(3);
```

Comment:
this change is because last verison of  stixCyberObservableEditField method anly checked  input[0] and allow wrong value for x_opencti_score like negative value or value > 100